tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -11349,11 +11408,11 @@ void hlsl::DiagnoseTranslationUnit(clang::Sema *self) {
     return;
   }
 
-  // Check RT shader if available for their payload use and match payload access
-  // against availiable payload modifiers.
-  // We have to do it late because we could have payload access in a called
-  // function and have to check the callgraph if the root shader has the right
-  // access rights to the payload structure.
+  // Check RT shader if available for their payload use and match payload
+  // access against availiable payload modifiers. We have to do it late
+  // because we could have payload access in a called function and have to
+  // check the callgraph if the root shader has the right access rights to
+  // the payload structure.
```

Comment:
Why the formatting changes in previously unmodified code?

Reply:
There are a ton of these, and it makes it hard to review the substantive changes.

Reply:
Fixed!